L1Trigger/TrackFindingTracklet/src/Stub.cc

```diff
@@ -122,6 +122,12 @@ std::string Stub::phiregionaddressstr() const {
   return phiregion.str() + stubindex_.str();
 }
 
+std::string Stub::phiregionstr() const {
+  int iphi = (phicorr_.value() >> (phicorr_.nbits() - settings_.nbitsallstubs(layerdisk())));
+  FPGAWord phiregion(iphi, 3, true, __LINE__, __FILE__);
```

Comment:
Magic numbers like "3" should be avoided, so try to replace this with a named constant sometime.